scripts/exgdas_enkf_ecen.sh

```diff
@@ -222,8 +222,13 @@ else
    LEVS_ENKF=${LEVS_ENKF:-$($NCLEN atminc_ensmean lev)} # get LEVS
 fi
 JCAP_ENKF=${JCAP_ENKF:--9999} # there is no jcap in these files
-[ $JCAP_ENKF -eq -9999 -a $LATB_ENKF -ne -9999 ] && JCAP_ENKF=$((LATB_ENKF-2))
-[ $LONB_ENKF -eq -9999 -o $LATB_ENKF -eq -9999 -o $LEVS_ENKF -eq -9999 -o $JCAP_ENKF -eq -9999 ] && exit -9999
+if [[ ${JCAP_ENKF} -eq -9999 && ${LATB_ENKF} -ne -9999 ]]; then
+   JCAP_ENKF=$((LATB_ENKF-2))
+fi
+if [[ ${LONB_ENKF} -eq -9999 || ${LATB_ENKF} -eq -9999 || ${LEVS_ENKF} -eq -9999 || ${JCAP_ENKF} -eq -9999 ]]; then
+   export err=9
```

Comment:
So `err_chk` needs `err=9` instead of `err=-9999` here?

Reply:
`-9999` would probably be OK, but I'm just not a fan of negative error codes.  I'll grit my teeth and write `-9999` if desired :smile:.

Reply:
No, that's fine lol! Just wanted to check 9 was fine. 😄